app/assets/javascripts/oxalis/view/right-menu/trees_tab_view.js

```diff
@@ -90,7 +91,8 @@ class TreesTabView extends React.PureComponent<Props, State> {
     // Wait for the Modal to render
     await Utils.sleep(1000);
     const state = Store.getState();
-    const nml = serializeToNml(state, this.props.skeletonTracing);
+    const buildInfo = await getBuildInfo();
```

Comment:
I'd suggest to load this from the server in parallel to sleeping one second (two lines above), like so:

```
const [__, buildInfo] = await Promise.all(Utils.sleep(1000), getBuildInfo());
```